gatsby-ssr.js

```diff
@@ -25,6 +31,9 @@ export const onRenderBody = ({ setHtmlAttributes, setHeadComponents, setPostBody
     />
   ]);
 
+  /* eslint-disable-next-line react/no-danger */
+  setPreBodyComponents([<script key="color-scheme-js" dangerouslySetInnerHTML={{ __html }} />]);
```

Comment:
This script ensures localStorage is primed with the correct `color-scheme` value and sets the HTML background color accordingly